src/components/item/publish/publicationButtons/ReadyToPublishButton.tsx

```diff
@@ -0,0 +1,88 @@
+import { LoadingButton } from '@mui/lab';
+import { Alert } from '@mui/material';
+
+import { PackedItem } from '@graasp/sdk';
+
+import { CheckIcon } from 'lucide-react';
+
+import useModalStatus from '@/components/hooks/useModalStatus';
+import { useBuilderTranslation } from '@/config/i18n';
+import { mutations } from '@/config/queryClient';
+import { buildItemPublicationButton } from '@/config/selectors';
+import { BUILDER } from '@/langs/constants';
+import { PublicationStatus } from '@/types/publication';
+
+import PublicVisibilityModal from '../PublicVisibilityModal';
+import PublicationButton from './PublicationButton';
+
+type Props = {
+  item: PackedItem;
+  isLoading: boolean;
+  notifyCoEditors: boolean;
+};
+
+const { usePublishItem } = mutations;
+
+export const ReadyToPublishButton = ({
+  item,
+  isLoading,
+  notifyCoEditors,
+}: Props): JSX.Element => {
+  const { t } = useBuilderTranslation();
+  const { id: itemId, public: isPublic } = item;
+  const { isOpen, openModal, closeModal } = useModalStatus();
+
+  const { mutate: publish, isLoading: isPublishing } = usePublishItem();
+
+  const publishItem = () =>
+    publish({ id: itemId, notification: notifyCoEditors });
+
+  const handlePublishItem = () => {
+    if (isPublic) {
+      publishItem();
+    } else {
+      openModal();
+    }
+  };
+
+  const handleModalValidate = () => {
+    publishItem();
+    closeModal();
+  };
+
+  const description = (
+    <Alert icon={<CheckIcon fontSize="inherit" />} severity="info">
```

Comment:
It bugs me that you use the `info` severity with the check ... why not use the `success` ?